--- components/omnibox/browser/base_search_provider.cc	2016-09-04 20:42:58.980243211 -0400
+++ components/omnibox/browser/base_search_provider.cc.new	2016-09-04 20:42:26.279452003 -0400
@@ -251,7 +251,7 @@
           base::CompareCase::SENSITIVE)) {
     match.inline_autocompletion =
         suggestion.suggestion().substr(input.text().length());
-    match.allowed_to_be_default_match = true;
+    match.allowed_to_be_default_match = false;
   }
   match.fill_into_edit.append(suggestion.suggestion());
 
--- components/omnibox/browser/search_provider.cc	2016-09-04 20:42:57.566875682 -0400
+++ components/omnibox/browser/search_provider.cc.new	2016-09-04 20:42:52.933430245 -0400
@@ -285,7 +285,7 @@
       match.contents_class.push_back(
           ACMatchClassification(0, ACMatchClassification::NONE));
       match.keyword = providers_.default_provider();
-      match.allowed_to_be_default_match = true;
+      match.allowed_to_be_default_match = false;
       matches_.push_back(match);
     }
     Stop(true, false);
